<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des livres</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
  border-image: url(://6bfc6c108c0779ba978a8cce9352aa2e.jpg);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1121</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Gestion des Livres</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-20</y>
        <width>1311</width>
        <height>521</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Afficher les livres</string>
       </attribute>
       <widget class="QTableView" name="tabpost">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>621</width>
          <height>131</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>240</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>salaire</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_grade">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>210</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_ajouter">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>180</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_salaire">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>240</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>210</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>grade</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>180</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>180</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
</string>
        </property>
        <property name="text">
         <string>Identifiant</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_supprimer">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>240</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_modifier">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>210</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_rechercher">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>150</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Rechercher</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>280</y>
          <width>69</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>competence</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_competences">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>280</y>
          <width>113</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_mission">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>330</y>
          <width>113</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>330</y>
          <width>69</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Mission</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>330</y>
          <width>112</width>
          <height>34</height>
         </rect>
        </property>
        <property name="text">
         <string>CLEAR</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>30</y>
          <width>112</width>
          <height>34</height>
         </rect>
        </property>
        <property name="text">
         <string>Tri</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>390</y>
          <width>112</width>
          <height>34</height>
         </rect>
        </property>
        <property name="text">
         <string>Show</string>
        </property>
       </widget>
       <widget class="QLabel" name="lblBkImge">
        <property name="geometry">
         <rect>
          <x>840</x>
          <y>100</y>
          <width>191</width>
          <height>351</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>400</y>
          <width>61</width>
          <height>25</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit{
background-color: transparent;
border-image: url(:off.png);
background: none;
border: none;
background-repeat: none;
}</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>430</y>
          <width>69</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>-90</y>
          <width>1101</width>
          <height>591</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(C:/Users/lenovo/Documents/2eme/2 tremestre/projet c++/projet2020/afficher5.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>label_11</zorder>
       <zorder>lineEdit</zorder>
       <zorder>tabpost</zorder>
       <zorder>label_3</zorder>
       <zorder>lineEdit_grade</zorder>
       <zorder>pb_ajouter</zorder>
       <zorder>lineEdit_salaire</zorder>
       <zorder>label_2</zorder>
       <zorder>lineEdit_id</zorder>
       <zorder>label</zorder>
       <zorder>pb_supprimer</zorder>
       <zorder>pb_modifier</zorder>
       <zorder>pb_rechercher</zorder>
       <zorder>label_4</zorder>
       <zorder>lineEdit_competences</zorder>
       <zorder>lineEdit_mission</zorder>
       <zorder>label_9</zorder>
       <zorder>pushButton</zorder>
       <zorder>pushButton_2</zorder>
       <zorder>pushButton_3</zorder>
       <zorder>lblBkImge</zorder>
       <zorder>label_10</zorder>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Gestion des Etagères</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>371</width>
        <height>251</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Ajouter une étagère</string>
       </attribute>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Identifiant</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_3">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>40</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_quantite">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>100</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Quantité</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_etat">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>160</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Etat</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_ajouter_2">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>200</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Afficher les étagères</string>
       </attribute>
       <widget class="QTableView" name="tabetagere">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>341</width>
          <height>171</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Supprimer une étagère</string>
       </attribute>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>60</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Identifiant</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_6">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>60</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_supprimer_2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>150</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_18">
       <attribute name="title">
        <string>Modifier une étagère</string>
       </attribute>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Identifiant</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_idmod_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>50</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>160</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Etat</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_etatmod">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>160</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_24">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>100</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Quantité</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_quantitemod">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>100</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_modifier_4">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>190</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_20">
       <attribute name="title">
        <string>Recherche une étagère</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_idmod_3">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>20</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_26">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>20</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Identifiant</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_modifier_5">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>50</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Recherche</string>
        </property>
       </widget>
       <widget class="QTableView" name="tabrecherche2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>80</y>
          <width>281</width>
          <height>131</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1119</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Gestion des livres</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
